vnext/docs/ParityStatus.md

```diff
@@ -0,0 +1,38 @@
+# Component & Module API Parity Status
+This doc reports the status progress of core components modules/APIs integration for vnext. The components and modules tracked here will be in alignment with the [React Native Lean Core](https://github.com/facebook/react-native/issues/23313) effort.
+
```

Comment:
Perhaps at the end of this list, add a little note on other APIs that are non- lean core but currently (partially/fully) supported and make a mention that we will be refactoring them into separate packages similar to the React Native Lean Core effort in Milestone 3.

Reply:
I added that at the top as a call out, and linked to the nvext issue submission page.